### ISSUE TITLE
[Bug] Crash when tapping settings after quickly enabling/disabling add-on

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#9647]
## Prerequisites
Have Search by Image installed.

## Steps to reproduce
1.Repeatedly diisable/enable Search by Image and quickly tap the Settings button.

### Expected behavior
The Settings are properly displayed.

### Actual behavior
Crash

### Device information
* Android device: Google Pixel 3a (Android 10)

* Fenix version: Custom Nightly(Build #20861010) 38.0.20200325130041, c580c2293
GV: 76.0a1-20200324093140 AS: 0.55.0

### Notes
►Video
![20200330-133238](https://user-images.githubusercontent.com/51314259/77903269-3e550000-728b-11ea-897c-91b79184b63e.gif)

►Log
```kotlin
--------- beginning of crash
2020-03-30 13:19:51.816 28818-28818/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.mozilla.fenix.nightly, PID: 28818
    kotlin.TypeCastException: null cannot be cast to non-null type kotlin.String
        at -$$LambdaGroup$js$yYMvyZVuCIBAx87qTqj6O0z70jU.onClick(java-style lambda group:27)
        at android.view.View.performClick(View.java:7259)
        at android.view.View.performClickInternal(View.java:7236)
        at android.view.View.access$3600(View.java:801)
        at android.view.View$PerformClick.run(View.java:27892)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
2020-03-30 13:19:51.817 28818-28818/? I/mozac/CrashReporter: Received crash: UncaughtExceptionCrash(throwable=kotlin.TypeCastException: null cannot be cast to non-null type kotlin.String, breadcrumbs=[Breadcrumb(message=Changing to fragment homeFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:18:52 GMT+03:00 2020), Breadcrumb(message=Changing to fragment settingsFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:18:55 GMT+03:00 2020), Breadcrumb(message=Changing to fragment addonsManagementFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:18:57 GMT+03:00 2020), Breadcrumb(message=Changing to fragment installedAddonDetailsFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:18:58 GMT+03:00 2020), Breadcrumb(message=Changing to fragment addonPermissionsDetailFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:14 GMT+03:00 2020), Breadcrumb(message=Changing to fragment installedAddonDetailsFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:15 GMT+03:00 2020), Breadcrumb(message=Changing to fragment addonDetailsFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:18 GMT+03:00 2020), Breadcrumb(message=Changing to fragment installedAddonDetailsFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:19 GMT+03:00 2020), Breadcrumb(message=Changing to fragment addonsManagementFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:20 GMT+03:00 2020), Breadcrumb(message=Changing to fragment addonDetailsFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:23 GMT+03:00 2020), Breadcrumb(message=Changing to fragment addonsManagementFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:24 GMT+03:00 2020), Breadcrumb(message=Changing to fragment addonDetailsFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:25 GMT+03:00 2020), Breadcrumb(message=Changing to fragment addonsManagementFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:25 GMT+03:00 2020), Breadcrumb(message=Changing to fragment installedAddonDetailsFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:38 GMT+03:00 2020), Breadcrumb(message=Changing to fragment addonDetailsFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:46 GMT+03:00 2020), Breadcrumb(message=Changing to fragment installedAddonDetailsFragment, isCustomTab: false, data={}, category=DestinationChanged, level=INFO, type=DEFAULT, date=Mon Mar 30 13:19:47 GMT+03:00 2020)])
2020-03-30 13:19:51.818 28818-28818/? I/mozac/CrashReporter: Showing notification

```

**Comment by Amejia481** [comment]
This should be fix, in master @AndiAJ could you take a look at it? :)

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on my build from master **`(Build #20980914)
38.0.20200406190117, 7b108961a GV: 76.0a1-20200406092400 AS: 0.55.0`** using the following devices:
• Google Pixel 3a (Android 10)
• Huawei Mate 20 Lite (Android 9)
• OnePlus A3 (Android 6.0.1)
